app/main.py

```diff
@@ -16,4 +16,7 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [marks, my_favourite_films, collection_of_coins],
+    "immutamle": [profile_info, lucky_number, name, pi, one_is_a_prime_number]
```

Comment:
There is a typo in the key name 'immutamle'. It should be 'immutable'. This is an error related to the task requirements as it can lead to confusion or bugs when accessing this dictionary.